lua/telescope/builtin/init.lua

```diff
@@ -913,10 +935,10 @@ builtin.tags = function(opts)
     },
     previewer = previewers.ctags.new(opts),
     sorter = conf.generic_sorter(opts),
-    attach_mappings = function(prompt_bufnr)
+    attach_mappings = function(_)
```

Comment:
you don't need any param here if you don't want. Only time to use `_` as a param is if you have some parameter you actually want later in the function. `function(_, _, param)` or something like that.